README.md

```diff
@@ -218,6 +218,30 @@ app({ model, view, update })
 [View online](http://codepen.io/jbucaran/pen/zNxRLy?editors=0010)
 </details>
 
+<details>
+<summary>JSX</summary>
+
+```jsx
+const view = (model, msg) => (
+    <div>
+        <button onclick={msg.add}>Add</button>
```

Comment:
is `onclick` valid in JSX? I thought it was `onClick`

If not, ignore my comment 😂